docs/usage/configuration.md

```diff
@@ -118,7 +118,15 @@ CLI arguments: `-d`, `--dry-run`
 
 The objective of the dry-run mode is to get a preview of the pending release. Dry-run mode skips the following steps: prepare, publish, success and fail. In addition to this it prints the next version and release notes to the console.
 
-**Note**: The Dry-run mode verifies the repository push permission, even though nothing will be pushed. The verification is done to help user to figure out potential configuration issues.
+**Note**: The Dry-run mode verifies the repository push permission, even though nothing will be pushed. The verification is done to help user to figure out potential configuration issues. If you do not want to verify push permissions nor push any changes use the [disable push](#disablepush) flag.
+
+### disablePush
+
+Type: `Boolean`<br>
+Default: `false`<br>
+CLI arguments: `--disable-push`
+
+Set to `true` to disable verifying push permissions and pushing anything. This can be useful when wanting to handle the release by yourself.
```

Comment:
> This can be useful when wanting to handle the release by yourself

what are you hoping to accomplish by "handling the release by yourself"? semantic-release is not a general purpose workflow tool. it is an opinionated release tool that is meant to manage the steps of releasing a version for you. if there is a desire to "handle the release yourself", semantic-release is likely not the appropriate tool for your goal

Reply:
Answered in: https://github.com/semantic-release/semantic-release/pull/1924#issuecomment-835769990